src/lib/seam/components/SupportedDeviceTable/SupportedDeviceRow.tsx

```diff
@@ -18,20 +19,30 @@ export function SupportedDeviceRow({
       <td>{deviceModel.main_category}</td>
       <td>{deviceModel.model_name}</td>
       <td>{deviceModel.manufacturer_model_id}</td>
-      <td>{deviceModel.connection_type}</td>
+      <td>{connectionTypeNames[deviceModel.connection_type]}</td>
       <td>
         <div
           className={classNames('seam-status-pill', `status-${statusColor}`)}
         >
-          <span>{deviceModel.support_level}</span>
+          <span>{capitalize(deviceModel.support_level)}</span>
         </div>
       </td>
     </tr>
   )
 }
 
-const supportLevelColors = {
-  Live: 'green',
-  Beta: 'blue',
-  Unsupported: 'orange',
+const supportLevelColors: Record<
+  DeviceModel['support_level'],
+  'green' | 'blue' | 'orange'
```

Comment:
Could be `string` but this shows we don't have type protection when mapping to dynamic class names.